modules/channel-messenger/package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "channel-messenger",
+  "fullName": "Channel Messenger",
   "version": "1.0.0",
-  "description": "Messenger Channel for Botpress",
+  "description": "A Botpress connector for Messenger",
```

Comment:
why don't we keep the channel naming ? we use channel everywhere. Module name, in the docs, even in the code

Reply:
Just so the description matches the description of other channels